Polish the doc (#13)
rhastie/easy-nmos: README.md

> # Easy-NMOS Docker Compose Solution

Reviewer: I agree with @pkeroulas, that even with a few "to dos" at the end of the page, you can lose the "INCOMPLETE INSTRUCTIONS" now, @rhastie :-)

Reviewer: Actually, I wonder whether "Docker Compose Solution" is necessary in the title?

> This starter kit allows the user to launch a simple NMOS setup with minimal installation steps. It is composed of 3 Docker containers:

Reviewer: ```suggestion
This starter kit allows the user to launch a simple NMOS setup with minimal installation steps. It is composed of three Docker containers:
```

> - a NMOS Registry (from [nmos-cpp](https://github.com/sony/nmos-cpp))
> - a Virtual NMOS Node (from [nmos-cpp](https://github.com/sony/nmos-cpp)) which should automatically register
> - a test suite (from [AMWA NMOS Testing Tool](https://github.com/AMWA-TV/nmos-testing))

Reviewer: ```suggestion
- an NMOS Registry (from [nmos-cpp](https://github.com/sony/nmos-cpp))
- a virtual NMOS Node (from [nmos-cpp](https://github.com/sony/nmos-cpp)) which should automatically register
- the [AMWA NMOS Testing Tool](https://github.com/AMWA-TV/nmos-testing)
```

All you need is a Linux host (Windows and Mac hopefully coming soon), with a recent version of [Docker](https://docs.docker.com/engine) and [Docker Compose](https://docs.docker.com/compose/) installed. Alternatively, you can run the implementation inside a Linux virtual machine.

## Brief installation instructions

### 0. Clone this repository locally or download and unpack the [archive](https://github.com/rhastie/easy-nmos/archive/master.zip)

### 1. Install Docker and Docker Compose

Follow the [short instructions](./install_docker.md).

### 2. Configure Compose IP addresses for your environment

> You will need 3 unused IP addresses that all reside in the same subnet as the host IP address. The docker-compose script by default starts all the containers with an external facing IP addresses so that they are easily accessible from your LAN.

Reviewer: ```suggestion
You will need 3 unused IP addresses that all reside in the same subnet as the host IP address. The docker-compose script by default starts all the containers with externally facing IP addresses so that they are easily accessible from your LAN.
```
